### PR TITLE
Paul abernathy

### PR DESCRIPTION
**Pull request by paulabernathy7**

### CODE REVIEW

**Comment by paulabernathy7** [comment]
MVP Complete

File: assignments/lambda-classes.js [hunk 1]
```diff
@@ -1 +1,119 @@
...
+
+class ProjectManger extends Instructor {
+    constructor(name, age, location, gender, gradClassName, favInstructor) {
+        super(name, age, location, gender)
+        this.gradClassName = gradClassName;
+        this.favInstructor = favInstructor
+    }
+
+    standup(channel) {
+        console.log( `${name} announces to ${channel}, @channel standy times!​​​​​`)
```
Comment (Dilstom) [inline assignments/lambda-classes.js:104]: do you mean `this.name` ? )

File: assignments/lambda-classes.js [hunk 2]
```diff
@@ -1 +1,119 @@
...
+    }
+}
+
+const jim = new ProjectManger('Jim', 28, 'Detroit', 'M', 'CS1', 'Sean')
+ 
+const adam = new ProjectManger('Adam', 25, 'Utah', 'M', 'CS1', 'Sean')
+
+jim.standup("webpt5")
+adam.debugsCode(paul, 'html')
+pam.grade(paul, 'python')
```
Comment (Dilstom) [inline assignments/lambda-classes.js:119]: this assignment looks good, I would encourage you to console.log objects you have been creating for better understanding 👍

File: assignments/prototype-refactor.js [hunk 3]
```diff
@@ -4,6 +4,184 @@ Prototype Refactor
...
+          this.team = obj.team;
+          this.weapons = obj.weapons;
+          this.langugage = obj.langugage;
+      }
+
+      greet() {
+        return `${this.name} offers a greeting in${this.langugage}.`
+
+      }
+  }
```
Comment (Dilstom) [inline assignments/prototype-refactor.js:110]: great!

**Review by Dilstom — APPROVED** [review]

**Pull request merged**